deepcell/metrics.py

```diff
@@ -727,7 +797,7 @@ def print_pixel_report(self):
         print('\nConfusion Matrix')
         print(self.cm)
 
-    def calc_object_stats(self, y_true, y_pred):
+    def calc_object_stats(self, y_true, y_pred, return_predictions=False):
```

Comment:
why don't we just always return the predictions?

Reply:
Right, I guess the question is what do you want the default behavior to be. You think it would be useful to always have this return? I was thinking that when you just want to print the metrics summary after training a new model you might not want anything to be returned? But I suppose it's equally easy to assign to _ if you don't want the output? Whatever you think makes more sense.

Reply:
You don't even need to assign it to anything if you don't want the return value.

To me, it just seems like unnecessary complexity. @msschwartz21 do you have any any thoughts on this?

Reply:
I would lean towards not returning it at all since someone can always use `self.predictions` if they want access to the data. Would the primary utility be for visualizing mistakes?

Reply:
Yes, the reason to return IDs would be for visualization or analysis using the label IDs

Reply:
I'm not sure whether or not any of the return is necessary, I'll leave that to you.

Whether or not we want to return data, we should always do it. let's remove the flag that controls whether or not something is returned.